packages/tasks/src/tasks/text-to-image/about.md

```diff
@@ -16,9 +16,25 @@ Different patterns can be generated to obtain unique pieces of fashion. Text-to-
 
 Architects can utilise the models to construct an environment based out on the requirements of the floor plan. This can also include the furniture that has to be placed in that environment.
 
-## Task Variants
+## Task Variants
 
-You can contribute variants of this task [here](https://github.com/huggingface/hub-docs/blob/main/tasks/src/text-to-image/about.md).
+### Image Editing
+
+Image editing with text-to-image models involves using text prompts to describe the wanted changes in an image and then follow them.
```

Comment:
```suggestion
Some text-to-image models can carry out modifications of an input image following the instructions provided in a text prompt.
```

Reply:
"some...can carry" - I feel like this part might be a bit misleading - as it's not an inherent capability of only some text to image models, wdyt?

Reply:
Sure, feel free to adjust!